medusa/show/history.py

```diff
@@ -110,18 +110,18 @@ def _get_actions(action):
 
         result = None
         if action == 'downloaded':
-            result = Quality.DOWNLOADED
+            result = DOWNLOADED
         elif action == 'snatched':
-            result = Quality.SNATCHED
+            result = SNATCHED
 
         return result or []
 
-    action_fields = ('action', 'provider', 'resource', 'date', 'proper_tags', 'manually_searched')
+    action_fields = ('action', 'quality', 'provider', 'resource', 'date', 'proper_tags', 'manually_searched')
     # A specific action from history
     Action = namedtuple('Action', action_fields)
     Action.width = len(action_fields)
 
-    index_fields = ('indexer_id', 'show_id', 'season', 'episode', 'quality')
+    index_fields = ('indexer_id', 'show_id', 'season', 'episode')
```

Comment:
As per the discussion on Slack, I think this is part of what affects the compact layout.

I'm against this change being made right now. I disagree with waiting for the Vue conversion to change the way it is displayed with this branch.
Please keep it as it is now on `develop`.

Reply:
Afaik the changes in history are all mako related. Changes in python are for separating status and quality

Reply:
Just based on the values here, I'm assuming it must have an effect on that.
This is exactly what defines a row in compact layout. Now that you removed `quality`, it's joining everything under one row.

I'm not 100% sure, but it looks like that to me, purely based on the Python code.

Reply:
Ill check